include/tc/core/execution_engine-inl.h

```diff
@@ -114,16 +114,16 @@ size_t ExecutionEngine<ExecutorType>::compile(
   return handle;
 }
 
-// Steal TcExecutor and give it back under lock
-// Run outside of lock on owning TcExecutor.
+// Steal ExecutorType and give it back under lock
```

Comment:
ExecutorType -> "executor" or "executor instance" in comments?   "Steal %templateArgumentName%" sounds like some crazy type manipulation.